features/distro-resources/src/main/resources/sitemaps/demo.sitemap

```diff
@@ -1,17 +1,91 @@
 sitemap demo label="Main Menu"
 {
 	Frame {
-		Group item=gFF label="First Floor" icon="firstfloor"
-		Group item=gGF label="Ground Floor" icon="groundfloor"
-		Group item=gC label="Cellar" icon="cellar"
-		Group item=Garden icon="garden"
+		Text label="First Floor" icon="firstfloor" {
+			Text label="Bathroom" icon="bath" {
+				Switch item=Light_FF_Bath_Ceiling label="Ceiling"
+				Switch item=Light_FF_Bath_Mirror label="Mirror"
+				Switch item=Heating_FF_Bath label="Heating"
+				Switch item=Shutter_FF_Bath label="Roller shutter"
+				Text item=Temperature_FF_Bath label="Temperature [%.1f °C]"
+				Text item=Window_FF_Bath label="Window [MAP(en.map):%s]"
+			}
+			Text label="Office" icon="office" {
+				Switch item=Light_FF_Office_Ceiling label="Ceiling"
+				Switch item=Heating_FF_Office label="Heating"
+				Switch item=Shutter_FF_Office_Window label="Window"
+				Switch item=Shutter_FF_Office_Door label="Door"
+				Text item=Temperature_FF_Office label="Temperature [%.1f °C]"
+				Text item=Window_FF_Office_Window label="Window [MAP(en.map):%s]"
+				Text item=Window_FF_Office_Door label="Balcony Door [MAP(en.map):%s]"
+				Text item=Wifi_Level label="Wifi Level [%d/4]"
+			}
+			Text label="Child's Room" icon="boy_1" {
```

Comment:
I would like to change that to an actuall common English name and maybe add a daughters room, too.

Reply:
sure, feel free.